### ISSUE TITLE
Usage survey settings refuse to save

### ISSUE THREAD

**Issue by MariusQuabeck** [nextcloud/survey_client#305]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

State of usage survey settings appear to not persist

### Steps to reproduce

1. goto /settings/admin/survey_client
2. uncheck everything and hit "Send new report now"
3. reload page

### Expected behavior

previous selection should be saved, instead every box is ticked again so I can seemingly not deactivate usage surveys

### Installation method

Official All-in-One appliance

### Nextcloud Server version

29

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.2

### Web server

Other

### Database engine version

PostgreSQL

### Is this bug present after an update or on a fresh install?

Upgraded to a MAJOR version (ex. 28 to 29)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by weinic** [comment]
I have the same problem.

I have had this problem since before NC v. 27
I noticed it at some point.

I am now on NC v. 31 and spoking the settings still does not work.


Is there a solution here? Is there any way to reset the "app" or remove the associated data from the database and reinitialize the app?

Maybe that will help....

**Comment by MariusQuabeck** [comment]
Funny timing — I just ran into this again and was about to open a new issue when I realized I already reported it last year 🙂

The behavior is still the same in AIO as described in the original report. I’ll keep an eye on things and check again once #374 lands in AIO.
